app/controllers/find_local_council_controller.rb

```diff
@@ -129,7 +129,14 @@ def fetch_location(postcode)
   end
 
   def setup_navigation_helpers
-    content_item = content_store.content_item(BASE_PATH)
-    @navigation_helpers = GovukNavigationHelpers::NavigationHelper.new(content_item)
+    @content_item = content_store.content_item(BASE_PATH).to_hash
+    # Remove the organisations from the content item - this will prevent the
```

Comment:
Would also be good to explain why we don't want the organisations meta tag in this application.